arch/arm/src/stm32h7/Kconfig

```diff
@@ -2086,6 +2138,15 @@ config STM32H7_QSPI_REGDEBUG
 
 endmenu
 
+config STM32H7_BYPASS_CLOCKCONFIG
+	bool "Bypass clock configuration"
+	depends on ARCH_STM32H7_DUALCORE
+	default n if ARCH_CHIP_STM32H7_CORTEXM7
+	default y if ARCH_CHIP_STM32H7_CORTEXM4
+	---help---
+		Bypass clock configuration. For dual core chips only one core
+		should configure clocks
+
```

Comment:
I'm a little confused by `STM32H7_BYPASS_CLOCKCONFIG`: I understand that only one core should configure clocks, but which core will do that? Maybe the `---help---` text could clarify that a little bit.

Reply:
This is the developer's decision. Both CPUs have RCC access. This is automatically selected so the dev doesn't have to do anything here. If someone wants to change the default behavior here, I assume they know exactly what they're doing

Reply:
Thanks.